assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -84,6 +96,7 @@ const BASE_INITIAL_STATE = {
 	// a module activation update, since the activation is technically complete
 	// before this data has been refreshed.
 	isAwaitingModulesRefresh: false,
+	registryKey: undefined,
```

Comment:
`registryKey` should be stored and retrieved from `core/site` now, so this isn't needed.

Reply:
Updated.